lib/tabular_form_builder.rb

```diff
@@ -70,24 +54,99 @@ def #{selector}(field, options = {})
   end
 
   def select(field, choices, options = {}, html_options = {})
-    label_for_field(field, options) + super
+    html_options[:class] = Array(html_options[:class]) + %w(form--select)
+
+    label_for_field(field, options) + container_wrap_field(super, 'select', options)
   end
 
   def collection_select(field, collection, value_method, text_method, options = {}, html_options = {})
-    label_for_field(field, options) + super
+    html_options[:class] = Array(html_options[:class]) + %w(form--select)
+
+    label_for_field(field, options) + container_wrap_field(super, 'select', options)
+  end
+
+  def collection_check_box(field,
+                           value,
+                           checked,
+                           text = field.to_s + "_#{value}",
+                           options = {})
+
+    label_options = options.reverse_merge(label: text)
+
+    label_for = "#{field}_#{value}".to_sym
+    label = label_for_field(label_for, label_options)
+
+    input_options = options.merge(multiple: true, no_label: true, checked: checked)
+    input = check_box(field, input_options, value, '')
+
+    label + container_wrap_field(input, 'check-box', options)
   end
 
   private
 
+  TEXT_LIKE_FIELDS = [
+    'number_field', 'password_field', 'url_field', 'telephone_field', 'email_field'
+  ].freeze
+
+  def container_wrap_field(field_html, selector, options = {})
+    ret = content_tag(:span, field_html, class: field_container_css_class(selector))
+    ret = content_tag(:span, ret, class: 'form--field-container') unless options[:no_label]
+
+    ret
+  end
+
+  def localize_field(field, options)
+    localized_field = Proc.new do |translation_form, multiple|
```

Comment:
Unused block argument - `multiple`. If it's necessary, use `_` or `_multiple` as an argument name to indicate that it won't be used.